options/locale/locale_en-US.ini

```diff
@@ -2446,6 +2446,13 @@ settings.thread_id = Thread ID
 settings.matrix.homeserver_url = Homeserver URL
 settings.matrix.room_id = Room ID
 settings.matrix.message_type = Message Type
+settings.visibility.private.button = Make Private
+settings.visibility.private.text = Changing the visibility to private will not only make the repo visible to only allowed members but may remove the relation between it and forks, watchers, and stars.
```

Comment:
IMO, It might be more understandable and user friendly to display this warning as bullet marks like:

**Changing the visibility will:**
- Make the repository only visible to members who have access
- Remove any affiliation between this repository and **forks**, **watchers**, and **stars**.

Reply:
Agree, but I think will need to change the [archive text](https://github.com/go-gitea/gitea/blob/main/options/locale/locale_en-US.ini#L2451) to bullet marks too.